django/core/cache/backends/base.py

```diff
@@ -208,9 +208,13 @@ def set_many(self, data, timeout=DEFAULT_TIMEOUT, version=None):
 
         If timeout is given, that timeout will be used for the key; otherwise
         the default cache timeout will be used.
+
+        Returns a list of keys that were not inserted, returns an empty list
```

Comment:
New docstrings should use PEP257 verb style "Returns -> Return". I'd write something like "On backends that supported it (memcached), return a list of keys that failed insertion."